ddtrace/contrib/langchain/patch.py

```diff
@@ -421,16 +429,19 @@ def traced_chat_model_generate(langchain, pin, func, instance, args, kwargs):
         span.finish()
         integration.metric(span, "dist", "request.duration", span.duration_ns)
         if integration.is_pc_sampled_log(span):
-            if chat_completions is None:
-                log_chat_completions = []
-            else:
-                log_chat_completions = [
-                    [
-                        {"content": message.text, "message_type": message.message.__class__.__name__}
-                        for message in messages
-                    ]
-                    for messages in chat_completions.generations
-                ]
+            log_chat_completions = []
```

Comment:
Logs are unofficially deprecated, I don't think we need to make corresponding changes to the logs code as we will be removing them soon.